provider/resource_replication.go

```diff
@@ -96,7 +96,6 @@ func resourceReplication() *schema.Resource {
 			"speed": {
 				Type:     schema.TypeInt,
 				Optional: true,
-				Default:  0,
```

Comment:
In the UI the default value is **-1**, should it also be here as well as default?